Source/lighting.cpp

```diff
@@ -58,9 +62,11 @@ const DisplacementOf<int8_t> VisionCrawlTable[23][15] = {
 	// clang-format on
 };
 
-uint8_t lightradius[16][128];
+/** 16 falloff tables for the light cone */
+uint8_t LightFalloffs[16][128];

```

Comment:
```suggestion
uint8_t LightFalloffs[LIGHTSIZE][128];
```

Reply:
I don't think there is a strict connection betten the number of light strengths and the number of light levels.

Reply:
See https://github.com/diasurgical/devilutionX/pull/6012/commits/af48fb234673545237df7fd3711f03f1fba161cc